### PR TITLE
feat(webapp): adjust and create branding possibilities

### LINKED ISSUE #5689: 🔧 [Refactor] Adjust design of toasters to the yunite style
## 🔧 Refactor ticket

Adjust the style of the toaster to the yunite design.

image
image

- [ ] Clarify the design colors etc
- [ ] Implement

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Adjust and create branding possibilities.

### Issues

Mainly:

- relates https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/issues/41

Under issues and PRs:

- fixes #5689
- relates https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/issues/33
- relates https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/issues/30
- relates https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/pull/31

### Todo

- [x] donation bar color, see https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/issues/33
- [x] see remainders of
  - [x] https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/issues/30
  - [x] https://github.com/Yunite-Net/Yunite-Net-Ocelot-Social-Deploy-Rebranding/pull/31
- [x] toaster colors https://github.com/ocelot-social-community/ocelot-social/issues/5689
- [x] color of newsfeed theme filter buttons on hover
- [x] adjust size of footer text to be as small as on yunite.org
  - [x] space between footer items
  - [x] size adjustment of footer text will be made on Yunite.org
- [x] group profile chips should be in Yunite primary color

image-3

### CODE REVIEW

File: webapp/pages/group/_id/_slug.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="isGroupVisible">
+  <div class="group-profile" v-if="isGroupVisible">
```
Comment (Elweyn) [inline webapp/pages/group/_id/_slug.vue:2]: What does this have to do with the branding changes?

**Review by Elweyn — COMMENTED** [review]
Seems good, just a question about the new class in group/_id/_slug.vue.

File: webapp/pages/group/_id/_slug.vue (hunk 1 above)
Comment (Tirokk) [reply webapp/pages/group/_id/_slug.vue:2]: It enables us to change the `ds-chip` background color in yunite just of this page.
